scripts/ZoneMinder/lib/ZoneMinder/Filter.pm

```diff
@@ -295,6 +300,10 @@ sub Sql {
             $self->{Sql} .= ' IN ('.join(',', @value_list).')';
           } elsif ( $term->{op} eq '!~' ) {
             $self->{Sql} .= ' NOT IN ('.join(',', @value_list).')';
+          } elsif ( $term->{op} eq 'LIKE' ) {
+            $self->{Sql} .= " LIKE $value";
```

Comment:
I would have expected value to need single quotes around it...

Reply:
The reason I didn't add single quotes is that line 244 puts quotes around each value.
https://github.com/ZoneMinder/zoneminder/blob/4b71bc75eab113240cf5a69650c2149839083ac4/scripts/ZoneMinder/lib/ZoneMinder/Filter.pm#L244

Reply:
you are of course correct